client/extensions/wp-super-cache/cdn-tab.jsx

```diff
@@ -79,7 +79,7 @@ const CdnTab = ( {
 								{ translate(
 									'The new URL to be used in place of %(url)s for rewriting. No trailing / please.',
 									{
-										args: { url: siteUrl },
+										args: { url: site && site.URL },
```

Comment:
I like to use lodash's [`get`](https://lodash.com/docs/4.17.4#get) -- `get( site, 'URL' )` in cases like this, which I find more semantic. No need to change it tho since it works fine as is.